test/cmdlineTests/standard_optimizer_yulDetails_optimiserSteps_mutliple_delimiters/input.json

```
@@ -1,7 +1,7 @@
 {
     "language": "Solidity",
     "sources": {
-        "A": {"urls": ["standard_optimizer_yulDetails_optimiserSteps_mutliple_delimiters/in.sol"]}
+        "A": {"urls": ["standard_optimizer_yulDetails_optimiserSteps_multiple_delimiters/in.sol"]}
```

Interesting that not updating the dir name did not make the test fail. That's because we validate the presence of sources but don't actually try to load them until all other options are validated.

We could just as well do this and remove the `in.sol` file:
```suggestion
        "A": {"content": ""}
```

We should do this in the other optimizer sequence tests as well to keep them simpler.